modules/core/src/v2/coins/algo.ts

```diff
@@ -55,10 +56,7 @@ interface TransactionExplanation extends BaseCoinTransactionExplanation {
   memo: string;
 }
 
-interface KeyPair {
-  pub: string;
-  prv: string;
-}
+const MAX_ALGORAND_NOTE_LENGTH = 1024;
```

Comment:
I think this was removed in another PR. Please remove if you don't intend on re-adding it.